src/core_plugins/kibana/public/management/sections/indices/create_index_pattern/create_index_pattern.js

```diff
@@ -30,15 +30,22 @@ uiModules.get('apps/management')
 
   // Configure the new index pattern we're going to create.
   this.formValues = {
-    id: $routeParams.id ? decodeURIComponent($routeParams.id) : undefined,
     name: config.get('indexPattern:placeholder'),
```

Comment:
I felt like it was more straightforward to have a block which describes a scenario where the defaults were changed than to use inline if's. I am fine changing it if you think it reads better as an inline if.